html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-fg.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<title>document wg color</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-fgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-linkcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-vlinkcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-alinkcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-bgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body text="red" linkColor="green" vlink="blue" alink="orange" bgColor="yellow">
+<script>
+test(function() {
+  assert_equals(document.fgColor, document.body.text);
+
+  assert_equals(document.linkColor, document.body.link);
+
+  assert_equals(document.vlinkColor, document.body.vLink);
+
+  assert_equals(document.alinkColor, document.body.aLink);
+
+  assert_equals(document.bgColor, document.body.bgColor);
+})
+test(function() {
+  document.body.text = red;
+
```

Comment:
Set this to “green”. And then check if it’s “green”. And do the samr for the rest.